src/validation.py

```diff
@@ -326,22 +321,19 @@ def validate_gross_to_net_conversion(cems, eia923_allocated):
     cems_net_not_equal_to_eia = validated_ng[validated_ng["pct_error"] != 0]
 
     if len(cems_net_not_equal_to_eia) > 0:
-        print(" ")
-        print(
-            f"WARNING: There are {len(cems_net_not_equal_to_eia)} plants where calculated annual net generation does not match EIA annual net generation."
+        logger.warning(" ")
+        logger.warning(
+            f"There are {len(cems_net_not_equal_to_eia)} plants where calculated annual net generation does not match EIA annual net generation."
         )
-        print(cems_net_not_equal_to_eia)
+        logger.warning("\n" + cems_net_not_equal_to_eia.to_string())
     else:
-        print("OK")
+        logger.info("OK")
```

Comment:
Not sure if printing these "OK" messages is necessary or not, or even if we need to print that we are running the test. Should we just print a warning if the warning is triggered? What do you think?

Reply:
I think have an explicit "this test was OK" is kind of nice, so I kept that